tests/lookup/tests.py

```diff
@@ -17,13 +17,20 @@ def setUp(self):
         self.au1 = Author.objects.create(name='Author 1')
         self.au2 = Author.objects.create(name='Author 2')
         # Create a couple of Articles.
-        self.a1 = Article.objects.create(headline='Article 1', pub_date=datetime(2005, 7, 26), author=self.au1)
-        self.a2 = Article.objects.create(headline='Article 2', pub_date=datetime(2005, 7, 27), author=self.au1)
-        self.a3 = Article.objects.create(headline='Article 3', pub_date=datetime(2005, 7, 27), author=self.au1)
-        self.a4 = Article.objects.create(headline='Article 4', pub_date=datetime(2005, 7, 28), author=self.au1)
-        self.a5 = Article.objects.create(headline='Article 5', pub_date=datetime(2005, 8, 1, 9, 0), author=self.au2)
-        self.a6 = Article.objects.create(headline='Article 6', pub_date=datetime(2005, 8, 1, 8, 0), author=self.au2)
-        self.a7 = Article.objects.create(headline='Article 7', pub_date=datetime(2005, 7, 27), author=self.au2)
+        self.a1 = Article.objects.create(headline='Article 1', pub_date=datetime(2005, 7, 26), author=self.au1,
+                                         slug='a1')
```

Comment:
@timgraham I'm slightly confused here, do you mean something like this?

```
self.a1 = Aritcle.objects.create(
   headline=...,
   pub_date=...,
   ...,
)
```

or something else?